components/modal/src/modal/modal.js

```diff
@@ -28,6 +28,27 @@ export const Modal = ({
     small,
 }) => {
     const layerStyles = resolveLayerStyles(hide)
+
+    useEffect(() => {
+        if (hide) {
+            return
+        }
+
+        const handleKeyDown = (event) => {
+            event.preventDefault()
```

Comment:
I'm not sure if `preventDefault` is necessary, I believe it's just for some HTML-components. Maybe @ismay has some input in regards to this?

Reply:
without the prevent default, keys like enter and space bar work as well, what I want is for escape key only to be able to close the modal.